src/javascript/app/common/account_opening.js

```diff
@@ -197,9 +197,11 @@ const AccountOpening = (() => {
     const handleNewAccount = (response, message_type) => {
         if (response.error) {
             const errorMessage = response.error.message;
+            const noticeBox    = $('#client_message').find('.notice-msg');
```

Comment:
for jQuery elements we use `$` in front of the name to make it obvious, and also the format should be lowercase letters separated by `_`, so `$notice_box`. could you also fix `error_message` above that? seems it was missed

Reply:
Roger!